src/gSP.cpp

```diff
@@ -524,7 +524,7 @@ void gSPLightVertexCBFD(u32 v, SPVertex * spVtx)
 			const f32 vy = (vtx.y + gSP.vertexCoordMod[9])*gSP.vertexCoordMod[13] - gSP.lights.pos_xyzw[l][Y];
 			const f32 vz = (vtx.z + gSP.vertexCoordMod[10])*gSP.vertexCoordMod[14] - gSP.lights.pos_xyzw[l][Z];
 			const f32 vw = (vtx.w + gSP.vertexCoordMod[11])*gSP.vertexCoordMod[15] - gSP.lights.pos_xyzw[l][W];
-			const f32 len = (vx*vx + vy*vy + vz*vz + vw*vw) / 65536.0f;
+			const f32 len = _FIXED2FLOAT((vx*vx + vy*vy + vz*vz + vw*vw),16);
```

Comment:
Purpose of _FIXED2FLOAT macro is to convert integer value to float. Here you use it to divide float value by 65536.
I know that result will be correct, but this use of that macro is confusing for code reader. 
Better use the same approach as in your previous commit:
f32 recip = FIXED2FLOATRECIP16;